### PR TITLE
[BEAM-2887] Remove special FnApi version of wordcount.

### PR DESCRIPTION
**Pull request by robertwb**
The "standard" wordcount now runs on FnApi, use that for integration testing.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by robertwb** [comment]
Run Python PostCommit

**Comment by robertwb** [comment]
R: @aaltay

File: sdks/python/apache_beam/examples/wordcount_it_test.py [hunk 1]
```diff
@@ -45,6 +44,13 @@ class WordCountIT(unittest.TestCase):
 
   @attr('IT')
   def test_wordcount_it(self):
+    self._run_wordcount_it()
+
+  @attr('IT', 'ValidatesContainer')
+  def test_wordcount_fnapi_it(self):
+    self._run_wordcount_it(experiment='beam_fn_api')
+
+  def _run_wordcount_it(self, **opts):
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/wordcount_it_test.py]: Where is `opts` is used? I am not sure how experiment is passed down.

Comment (robertwb) [reply sdks/python/apache_beam/examples/wordcount_it_test.py]: Oh, yes, this was completely lost in the refactor. Thanks for catching this!

**Comment by robertwb** [comment]
Run Python PostCommit

**Comment by robertwb** [comment]
Jeknins: Run Python PostCommit

**Comment by robertwb** [comment]
Jenkins: Run Python PostCommit

**Comment by robertwb** [comment]
The postcommit error seems to be for the unrelated test_user_score_it. Trying again.

Run Python PostCommit

**Comment by robertwb** [comment]
Run Python PostCommit

**Comment by robertwb** [comment]
Run Python PostCommit

**Pull request merged**